components-core/src/main/java/com/airwallex/android/core/Airwallex.kt

```diff
@@ -348,6 +422,32 @@ class Airwallex internal constructor(
         return response
     }
 
+    /**
+     * Retrieve available payment methods
+     *
+     * @param params [RetrieveAvailablePaymentMethodParams] used to retrieve all [AvailablePaymentMethodType]
+     * @param session The [AirwallexSession] which contains session information for retrieving payment methods.
+     * @param params [RetrieveAvailablePaymentMethodParams] Parameters used to retrieve all [AvailablePaymentMethodType].
+     */
+    fun retrieveAvailablePaymentMethodsAsync(
```

Comment:
We can use same name `retrieveAvailablePaymentMethods` as suspend one since function supports overwrite.